### ISSUE TITLE
create health impact summary file

### ISSUE THREAD

**Issue by lkoolik** [echo-air-model/echo-air#50]
Realized while putting together test cases that we never actually create a summary file with all of the health outcomes. I'm thinking something that looks like exposure_disparity.csv that has health outcomes by cause (e.g., IHD, lung cancer, all-casue) by race summed across the entire domain. Is this something you can create? It might actually be related to the aggregation issue.

**Comment by lkoolik** [comment]
@megha5219 @benjaminsalop - this is lower priority to the ISRM update, but could one of you please look into this once the ISRM stuff is done? It should be independent
